db/seeds.rb

```diff
@@ -1,9 +1,52 @@
 # frozen_string_literal: true
 
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# This file should contain all the record creation needed to seed the database with its default
+# values. The data can then be loaded with the rails db:seed command (or created alongside the
+# database with db:setup).
+
+a = Applicant.create(
+  first_name: 'George',
+  last_name: 'Wu',
+  utorid: 'wugeorge',
```

Comment:
What's the difference between utorid and student number?

Reply:
both are unique, but i presume HR wants both

Reply:
Well I meant that something like wugeorge vs ########. I hope both are unique. xD